README.md

```diff
@@ -34,81 +37,126 @@
-- Train a GPT-2 125M to a perplexity of 23.9 for $148 (_with vanilla PyTorch_: $255)
-- Use start-of-the-art implementations of methods to speed up your own training loop.
+### The Latest Research on Efficient Training
 
-At MosaicML, we are focused on making training ML models accessible. To do this, we continually productionize state-of-the-art academic research on efficient model training, and also study the _combinations_ of these methods in order to ensure that model training is ✨ as efficient as possible ✨.
+At MosaicML, our goal is to make training state-of-the-art deep learning models accessible to everyone.
+We do so through Composer:
 
-Everyone has their own priorities: best accuracy, cheapest cost, and somewhere in between.  Composer provides novel recipes that push the boundary of both cost and accuracy. Composer allows you to choose **the best model for your real-world constraints**.
+* We continually integrate <b>state-of-the-art research on efficient neural network training</b> (and our improvements to that research).
+* These speedup methods work on our <b>efficient implementations of up-to-date models and datasets</b>.
+* We provide <b>curated recipes of speedup methods</b> that lead to the most efficient possible training on these benchmarks.
 
-Composer features:
-- 20+ efficient training methods for training a better language and vision models! Don't waste hours trying to reproduce research papers when Composer has done the work for you.
-- Easy-to-use (optional) Trainer interface written to be as performant as possible, and [integrated best practices](https://www.mosaicml.com/blog/5-best-practices-for-efficient-model-training).
-- Easy-to-use Functional forms that allow you to integrate efficient training methods into your training loop!
-- Strong, _reproducible_ baselines to get you started as 💨 fast 💨 as possible
 
-## Quickstart
+## 📣 Quickstart
 
 ### Installation
-Composer is available with Pip
+Composer is available with Pip:
 
 ```bash
 pip install mosaicml
 ```
-Alternatively install Composer with Conda
+Alternatively, install Composer with Conda:
+
 ```bash
 conda install -c mosaicml mosaicml
 ```
 ---
 
 ### Usage
 
-Composer provides both a **Functional API** (similar to `torch.nn.functional`) and a **Trainer** (that abstracts away the training loop) to provide flexibility to users.
+You can use Composer's speedup methods in two ways:
+
+* Through a standalone **Functional API** (similar to `torch.nn.functional`) that allows you to integrate them into your existing training code.
+* Using Composer's built-in **Trainer**, which is designed to be performant and automatically takes care of many of the low-level details of using speedup methods.
 
 #### Example: Functional API [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mosaicml/composer/blob/dev/notebooks/Composer_Functional.ipynb)
```

Comment:
maybe move the "open in colab" button to a new line, immediately below the header?